pkgs/applications/misc/curaengine/stable.nix

```diff
@@ -13,6 +13,7 @@ stdenv.mkDerivation {
 
   postPatch = ''
     sed -i 's,--static,,g' Makefile
+    ${lib.optionalString stdenv.isi686 "sed -i 's,-flto,,g' -i Makefile"}
```

Comment:
Can we use substituteInPlace instead? It notified us if the pattern no longer matches.

Reply:
Thanks, done.